v2/poll_binding_last_operation_test.go

```diff
@@ -0,0 +1,209 @@
+package v2
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+)
+
+func defaultBindingLastOperationRequest() *BindingLastOperationRequest {
+	return &BindingLastOperationRequest{
+		InstanceID: testInstanceID,
+		BindingID:  testBindingID,
+		ServiceID:  strPtr(testServiceID),
+		PlanID:     strPtr(testPlanID),
+	}
+}
+
+const successBindingLastOperationRequestBody = `{"service_id":"test-service-id","plan_id":"test-plan-id"}`
+
+func TestPollBindingLastOperation(t *testing.T) {
+	cases := []struct {
+		name                string
+		enableAlpha         bool
+		originatingIdentity *OriginatingIdentity
```

Comment:
I am still highly dubious about sending originating identity with last operation requests, but that's not an issue of this PR.

Reply:
Noted.

Reply:
#85